mobile-app/src/atoms/atoms.ts

```diff
@@ -255,3 +258,32 @@ export const deleteNotificationAtom = atomFamily((id: number) =>
     },
   ),
 );
+
+// friend search
+export const searchQueryAtom = atom("", (_get, set, newValue: string) => {
+  set(searchQueryAtom, newValue);
+});
+
+export const searchResultsAtom = atom(async (get) => {
+  console.log("searching for friends");
+  return await searchForFriendsInDB(get(searchQueryAtom));
+});
+
```

Comment:
Would probably be better to have previously gotten all our friends from the DB. And then just search the local copy. No?

Reply:
Also remove console log for PR